architecture/core_objects/overview.adoc

```diff
@@ -5,7 +5,20 @@
 :icons:
 :experimental:
 
-If you'd like to contribute to OpenShift documentation, see our
-https://github.com/openshift/openshift-docs[source repository] and
-https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc[guidelines]
-to get started.
+The following sections give some information about general OpenShift and
+Kubernetes architecture and terms. The
+link:../architecture/core_objects/kubernetes_model.html[Kubernetes Model]
+section outlines how the Kubernetes infrastructure is used in your OpenShift
+instance. The link:../architecture/core_objects/openshift-model.html[OpenShift
+Model] section outlines any OpenShift-specific information about your
+infrastructure.
+
+The link:../architecture/core_objects/builds.html[Builds] section outlines how
+to create a working image from a build object, and the
+link:../architecture/core_objects/routing.html[Routing] seciton outlines the
+routing process.
+
+A number of Kubernetes objects can be created by utilizing templates.
+
+See the link:../dev_guide/templates.html[Templates] section for more
+information.
```

Comment:
You can just move this link in-line to the above sentence (also a minor rewrite suggestion/idea here):

"You can create many of these resources at once by utilizing link:../dev_guide/templates.html[templates]."